commands/DevicesCommand.js

```diff
@@ -41,9 +42,14 @@ class DevicesCommand extends BarracksCommand {
   }
 
   configureCommand(program) {
-    return program
-      .option('--segment [segmentName]', '(Optionnal) Render devices from that segment only (Cannot be used with --filter).')
-      .option('--filter [filterName]', '(Optionnal) Apply a filter on the device list (Cannot be used with --segment).');
+    if (config.experimental) {
```

Comment:
Shouldn't we add a private flag 'experimentalEnabled' or such that would be initialized in the BarracksCommand constructor, instead of importing config in all command where we need to check that flag ?